Extension/package.json

```diff
@@ -698,7 +697,7 @@
             ]
           },
           "description": "%c_cpp.configuration.commentContinuationPatterns.description%",
-          "scope": "resource"
+          "scope": "window"
```

Comment:
I'm not sure why these are showing up in your branch.  I thought @Colengms did that in another PR.